tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -264,6 +271,56 @@ RHODS Dahsboard Pod Should Contain OauthProxy Container
         List Should Contain Value    ${container_name}    oauth-proxy
     END
 
+Is Environment Staging
+    [Documentation]    Return true if environment is staging otherwise fails
+    ${list_of_links} =    Get Links From Switcher
+    Should Contain    ${list_of_links}[0]   devshift.org
+
+Check Application Switcher Links To Openshift Cluster Manager
+    ${ocm_staging_link} =    Set Variable    https://qaprodauth.cloud.redhat.com/openshift/details/
+    ${cluster_id} =    Get Cluster ID
+    ${cluster_id} =    Remove String    ${cluster_id}    "
+    ${list_of_links} =    Get Links From Switcher
+    ${status}    Run Keyword And Return Status    Is Environment Staging
+    IF    "${status}" == "True"
+        ${ocm_staging_link} =    Set Variable    ${ocm_staging_link}${cluster_id}
+        Log    ${ocm_staging_link}
+        Check HTTP Status Code    ${ocm_staging_link}    verify_ssl=${False}
+        Go To    ${ocm_staging_link}
+        ${cluster_name} =    Get Cluster Name By Cluster ID    ${cluster_id}
+        Login To OCM
+        Wait Until Page Contains    ${cluster_name}
```

Comment:
you should do the same checks for production environment

Try to be more specific in `Wait Until Page Contains ${cluster_name}`. Maybe checking the particular `<h1>` html tag or ther parts of the configuration page. Otherwise this kw may pass even if the user is redirected to homepage instead of cluster configuration page

Reply:
Done